dev_requirements.txt

```diff
@@ -13,3 +13,4 @@ pytz
 tox==3.25.1
 twine
 wheel
+-e .
```

Comment:
Please drop this commit (discussed offline).

Reply:
Done

Reply:
Ping